features/cellular/framework/AT/AT_CellularContext.cpp

```diff
@@ -407,23 +404,19 @@ bool AT_CellularContext::get_context()
 
 bool AT_CellularContext::set_new_context(int cid)
 {
-    bool modem_supports_ipv6 = get_property(PROPERTY_IPV6_PDP_TYPE);
-    bool modem_supports_ipv4 = get_property(PROPERTY_IPV4_PDP_TYPE);
-    bool modem_supports_nonip = get_property(PROPERTY_NON_IP_PDP_TYPE);
-
     char pdp_type_str[8 + 1] = {0};
     pdp_type_t pdp_type = IPV4_PDP_TYPE;
 
-    if (_nonip_req && _cp_in_use && modem_supports_nonip) {
+    if (_nonip_req && _cp_in_use && get_property(PROPERTY_NON_IP_PDP_TYPE)) {
         strncpy(pdp_type_str, "Non-IP", sizeof(pdp_type_str));
         pdp_type = NON_IP_PDP_TYPE;
-    } else if (modem_supports_ipv6 && modem_supports_ipv4) {
+    } else if (get_property(PROPERTY_IPV4V6_PDP_TYPE)) {
```

Comment:
I guess it's good to add here that IPV4V6, but should also be IPV4 & IPV6 for single stack support.

Reply:
If a target doesn't mark as supported the IPV4V6 feature but it has IPV4 and IPV6 support, I think we should consider it as it doesnt support IPV4V6. Why else wouldnt have it set accordingly?